test/testdata/data-index_and_job-service/dev/postgreSQL/01-postgres.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
```

Comment:
Can't we reuse this same postgres manifest?

Reply:
Sure. I'll rearrange the manifests

Reply:
I can't move this file to a common folder because kustomize doesn't allow me to reference maniifests that are in the not in the same folder as the `kustomization.yaml` file or in a subfolder to it.